src/leap/bitmask/services/mail/imap.py

```diff
@@ -63,9 +63,13 @@ def start_imap_service(*args, **kwargs):
     from leap.bitmask.config import flags
     logger.debug('Launching imap service')
 
+    # TODO --------------------------------------------
+    # this does not make sense in imap anymore, should
+    # be moved to incoming_mail_service
     override_period = get_mail_check_period()
     if override_period:
         kwargs['check_period'] = override_period
+    # -------------------------------------------------
```

Comment:
This has to be moved to `start_incoming_mail_service` my bad. I'll pull-req on this pull-req to fix it.